### ISSUE TITLE
Config file not found on Android

### ISSUE THREAD

**Issue by c100k** [bugsnag/bugsnag-dsym-upload#10]
### Expected behavior

Indicate a specific when the config file is not found.

### Observed behavior

> A Bugsnag API key is required to release a build. Set com.bugsnag.android.API_KEY in your AndroidManifest.xml to detect API key automatically.

### Steps to reproduce

It is a React Native project organized like this :

```
/android
/ios
/fastlane
   /Fastfile
```

#### With no config_file

```ruby
platform :android do
    lane :beta do
        send_build_to_bugsnag
    end
end
```

> A Bugsnag API key is required to release a build. Set the config_file option with the path to your AndroidManifest.xml and set com.bugsnag.android.API_KEY in it to detect API key automatically.

#### With good config_file containing the API KEY

```ruby
platform :android do
    lane :beta2 do
        send_build_to_bugsnag(
            config_file: "./android/app/src/main/AndroidManifest.xml",
        )
    end
end
```

> A Bugsnag API key is required to release a build. Set com.bugsnag.android.API_KEY in your AndroidManifest.xml to detect API key automatically.

#### With unknown config_file

```ruby
platform :android do
    lane :beta2 do
        send_build_to_bugsnag(
            config_file: "./android/app/src/main/UnknownFile.xml",
        )
    end
end
```

> A Bugsnag API key is required to release a build. Set com.bugsnag.android.API_KEY in your AndroidManifest.xml to detect API key automatically.

There is no way to figure out what's wrong.

### Version

```
+------------------------+---------+------------------------+
|                       Used plugins                        |
+------------------------+---------+------------------------+
| Plugin                 | Version | Action                 |
+------------------------+---------+------------------------+
| fastlane-plugin-bugsn  | 1.2.1   | send_build_to_bugsnag  |
| ag                     |         | upload_symbols_to_bug  |
|                        |         | snag                   |
+------------------------+---------+------------------------+
```

#### Can't comment on Issues?
Some users have been unable to comment on Github issues when an [adblocker extension is enabled](https://docs.bugsnag.com/platforms/browsers/faq/#is-bugsnag-blocked-by-ad-blockers).
We recommend temporarily disabling the extension, or if that fails, contacting support@bugsnag.com.

**Comment by kattrali** [comment]
Thanks for the report, @c100k. What is the output of `fastlane action send_build_to_bugsnag`? Are you running the command from the root of your project?

**Comment by c100k** [comment]
Hi @kattrali,

Here is the output of `fastlane action send_build_to_bugsnag`.

```
+------------------------+---------+------------------------+
|                       Used plugins                        |
+------------------------+---------+------------------------+
| Plugin                 | Version | Action                 |
+------------------------+---------+------------------------+
| fastlane-plugin-incre  | 0.4.3   | increment_version_cod  |
| ment_version_code      |         | e                      |
| fastlane-plugin-bugsn  | 1.2.1   | send_build_to_bugsnag  |
| ag                     |         | upload_symbols_to_bug  |
|                        |         | snag                   |
+------------------------+---------+------------------------+

Loading documentation for send_build_to_bugsnag:

+---------------------------------------------------------------------------+
|                           send_build_to_bugsnag                           |
+---------------------------------------------------------------------------+
| Notifies Bugsnag of a build                                               |
|                                                                           |
| Notifies Bugsnag of a new build being released including app version and  |
| source control details                                                    |
|                                                                           |
| Created by cawllec                                                        |
+---------------------------------------------------------------------------+

+------------------+------------------+---------+------------------+
|                  send_build_to_bugsnag Options                   |
+------------------+------------------+---------+------------------+
| Key              | Description      | Env Var | Default          |
+------------------+------------------+---------+------------------+
| config_file      | AndroidManifest  |         |                  |
|                  | .xml/Info.plist  |         |                  |
|                  | location         |         |                  |
| api_key          | Bugsnag API Key  |         |                  |
| app_version      | App version      |         |                  |
|                  | being built      |         |                  |
| android_version  | Android app      |         |                  |
| _code            | version code     |         |                  |
| ios_bundle_vers  | iOS/macOS/tvOS   |         |                  |
| ion              | bundle version   |         |                  |
| release_stage    | Release stage    |         | production       |
|                  | being built,     |         |                  |
|                  | i.e. staging,    |         |                  |
|                  | production       |         |                  |
| builder          | The name of the  |         | chafikhnini      |
|                  | entity           |         |                  |
|                  | triggering the   |         |                  |
|                  | build            |         |                  |
| repository       | The source       |         | git@github.com:  |
|                  | control          |         | intra-vp/BugsBu  |
|                  | repository URL   |         | nny.git          |
|                  | for this         |         |                  |
|                  | application      |         |                  |
| revision         | The source       |         | d62250a1b3a1ec0  |
|                  | control          |         | 255d3dfa744d07c  |
|                  | revision id      |         | 0ec86c9555       |
| provider         | The source       |         |                  |
|                  | control          |         |                  |
|                  | provider, one    |         |                  |
|                  | of               |         |                  |
|                  | 'github-enterpr  |         |                  |
|                  | ise',            |         |                  |
|                  | 'gitlab-onpremi  |         |                  |
|                  | se', or          |         |                  |
|                  | 'bitbucket-serv  |         |                  |
|                  | er', if any      |         |                  |
| endpoint         | Bugsnag          |         | https://build.b  |
|                  | deployment       |         | ugsnag.com       |
|                  | endpoint         |         |                  |
+------------------+------------------+---------+------------------+

More information can be found on https://docs.fastlane.tools/actions/send_build_to_bugsnag
```

And yes, I'm running the commands in the root of the project.

Thanks,

**Comment by kattrali** [comment]
Thanks @c100k. Is this the output when overriding `config_file`?

**Comment by c100k** [comment]
@kattrali What do you mean ? This command is only supposed to give me the documentation, right ?

**Comment by kattrali** [comment]
It took a bit of debugging but there were a few issues in the way of using the plugin from a React Native project. These should be resolved now in [v1.3.0](https://github.com/bugsnag/bugsnag-dsym-upload/releases/tag/v1.3.0).

**Comment by c100k** [comment]
I confirm, it works.
Thanks for fixing this so quickly !
